booking/src/main/java/org/dcsa/conformance/standards/booking/action/BookingAction.java

```diff
@@ -24,7 +24,7 @@
 public abstract class BookingAction extends ConformanceAction {
   protected final int expectedStatus;
   private final OverwritingReference<DynamicScenarioParameters> dspReference;
-
+  protected boolean assignCbr;
```

Comment:
Not used. If used later, it should most likely be final.

Reply:
Scratch that - that attribute does not belong in this layer. Not all booking actions can assign a CBR